### DISCUSSION TITLE
Ability to hide certain configs (via airbyte-hidden) dynamically based on environment using environment variable

### DISCUSSION THREAD

**Discussion by abhijithcdreamorbit** [airbytehq/airbyte#33752] [Connector Questions]
Does airbyte support hiding configs based on environments i.e env variable?
Like for instance. 
Spec json would look like
{
  "some_key": {
    "type": "string",
    "title": "Some Key"
  },
  "use_fake_api": {
    "type": "boolean",
    "title": "To use mocked api's",
    "airbyte_hidden": true
  }
}
Idea is to have a way to dynamically hide use_fake_api for production env
Currently airbyte-hidden excepts only true or false. Doesn't allow to set to an evaluated value

**Comment by marcosmarxm** [comment]
The answer is no. What configs do you want to hide? Can you update the title of your discussion it doesn't give any meaningful context about the problem or feature you want to discuss.

**Reply by abhijithcdreamorbit** [reply]
Have updated it

[a comment by an automated account is not shown]
